docs/pi_setup.md

```diff
@@ -1,35 +1,60 @@
-# Setting up dora on the PI
+# Setting up dora on the Pi
 
-You need a pi3 or later with onboard WIFI module (or an external WIFI dongle) and raspbian (I'm running 32-bit)
+You will need a Pi3 or later with either an onboard WiFi module or an external WiFi dongle. We also assume that you are running Raspbian (tested with 32-bit).
 
-1.  SSH into the pi
+### 0. SSH into the Pi
 
+Ensure everything is up to date with:
+
+```bash
+sudo apt update
+sudo apt full-upgrade
 ```
+
+Then ensure you acquire the prerequisites:
+
+```bash
 sudo apt-get -y install hostapd bridge-utils iptables gettext libdbus-1-dev libidn11-dev libnetfilter-conntrack-dev nettle-dev netfilter-persistent iptables-persistent
 ```
 
-## set up the pi as an access point
+Also, if you have not yet installed Rust on your Pi, this can be achieved rather painlessly:
 
-I followed the guide [here](https://www.raspberrypi.com/documentation/computers/configuration.html#setting-up-a-routed-wireless-access-point)
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Furthermore, you may wish to also install `docker` if you have not yet done so:
```

Comment:
Docker is necessary only for cross compiling with `cross` so maybe we should put this in the main README under that section

Reply:
I agree that this is probably the better way to go 👍